styles.css

```diff
@@ -1,138 +1,601 @@
+  width: 100%;
+  height: 10vh;
+  padding: 3%;
+  top: 0;
 }
 
-nav a {
-  padding-right: 3%;
+.list-nav {
+  display: none;
 }
 
-.logo a {
-  font-size: 1.2rem;
-  font-weight: 700;
+.link {
+  margin-bottom: 3.4rem;
 }
 
-.link nav a {
+a {
   text-decoration: none;
-  display: inline-block;
-  flex-direction: row;
+}
+
+.logo a {
+  font-size: 1.2rem;
+  font-weight: 700;
 }
 
 .logo {
-  margin-top: 20px;
   color: #6070ff;
-  font-size: 16px;
+  font-size: 1.8rem;
+  font-weight: 700;
+  text-decoration: none;
 }
 
-.nav-bar .logo {
-  flex-wrap: wrap;
-  flex: 1;
-  order: none;
-  margin-left: 10px;
+.container-all {
+  display: flex;
+  flex-direction: column;
+  background-image: url('./images/header.now.png');
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: 100% 9rem;
+  padding-bottom: 1rem;
+  margin-bottom: 12%;
+  background-color: #ffff;
+  border-bottom-left-radius: 4pc;
 }
 
-.dashboard {
-  border: 0 solid #acb7c3;
-  color: #6070ff;
-}
+.works {
+  display: grid;
+  grid-template-columns: 1fr;
+  gap: 12.7rem;
+  background-color: #f7f7f7;
+  padding: 12.8rem 0 2.8rem 0;
 
-.nav-bar .dashboard {
-  margin-right: 10px;
-  margin-top: 20px;
+  /* padding-bottom: 12.8rem; */
+  margin-bottom: 12.8rem;
 }
 
-.container {
+.container1 {
   display: flex;
-  flex-wrap: wrap;
   flex-direction: column;
-  background-image: url('./images/header-shapes\ mobile.png');
+  margin-top: 10%;
+  margin-bottom: 0%;
+  background-size: cover;
   background-repeat: no-repeat;
-  background-size: cover; 
-  justify-content: center;
-  
+  background-position: 100%;
+  background-image: url('./images/header-shapes\ mobile.png');
 }
 
-h1 {
-  margin: 0;
-  padding-top: 10px;
-  margin-top: 32%;
-  overflow: hidden;
+.iam {
   font-style: normal;
-  font-size: 32px;
-  text-transform: capitalize;
-  line-height: 42px;
+  font-weight: 700;
+  font-size: 48px;
+  line-height: 60px;
   color: #172b4d;
+  text-transform: capitalize;
+  letter-spacing: 0.37px;
+  margin-top: 0;
+  padding: 0;
 }
 
-.intro > p {
-  text-align: left;
-  left: 10px;
-  margin-top: 0;
-  font-size: 14.3px;
+.iam span {
+  display: none;
+}
+
+.firstp {
   color: #344563;
+  font-weight: 400;
+  font-style: normal;
+  font-size: 16px;
+  line-height: 24px;
+  padding-bottom: 1rem;
 }
 
-h2 {
-  color: #7f8ccf;
+.connect1 {
+  font-style: normal;
+  font-weight: 500;
   font-size: 16px;
-  line-height: 1.3em;
-  font-weight: 400;
+  line-height: 24px;
+  color: #7f8cff;
+}
+
+.connect {
   font-style: normal;
-  top: 20%;
-  bottom: 20%;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  color: #091e42;
+}
+
+.lets-connect {
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  color: #091e42;
+  margin-bottom: 1.2rem;
 }
 
 .media-icons {
   list-style-type: none;
   display: flex;
-  flex-wrap: wrap;
-  padding-inline-start: 0;
+  gap: 2rem;
+  align-items: center;
+}
```

Comment:
- [ ] Please add a bit more margin between the `let's connect` heading and  social media icon to match the provided [design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=0-1&t=UQqckzz8N6Oyx5Mf-0). Also I would suggest you to remove margin from the section instead use padding since the background in the design doesn't seem to have a margin.

|Figma design |Your design|
|---|----|
|<img width="322" alt="Screenshot 2023-05-29 at 10 33 17 PM" src="https://github.com/hetrox8/portfolio1/assets/44798044/2b1e6dc8-bacb-4707-98e1-4c1ca22d81c9">|<img width="366" alt="Screenshot 2023-05-29 at 10 33 43 PM" src="https://github.com/hetrox8/portfolio1/assets/44798044/95219daf-dcb5-4cfd-bba4-4936f411ed5a">|